### DISCUSSION TITLE
OneSignal Delivery Method

### DISCUSSION THREAD

**Discussion by Spone** [excid3/noticed#473] [Ideas]
Hi there!

Thanks for a very useful gem!

I'm posting here to discuss potential interest in building a Delivery Method for OneSignal, as I may have a use case for it soon.

What would be your preferred approach? Implement using the [official Ruby client](https://documentation.onesignal.com/docs/ruby-client-sdk) or implement from scratch?

Thanks!

**Comment by excid3** [comment]
Probably the official gem if it's well maintained.

**Reply by Spone** [reply]
![image](https://github.com/user-attachments/assets/ad4059c8-6ba0-4aa0-91fb-95b1063ed452)

🙄

**Reply by excid3** [reply]
🤦‍♂️
